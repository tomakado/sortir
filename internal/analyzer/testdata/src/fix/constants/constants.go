package constants

const (
	C = 3
	A = 1 // want "variable/constant declarations are not sorted"
	B = 2
)

const (
	Z = "z"
	X = "x" // want "variable/constant declarations are not sorted"
	Y = "y"
)