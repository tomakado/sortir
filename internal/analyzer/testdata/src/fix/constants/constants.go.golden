package constants

const (
	A = 1 // want "variable/constant declarations are not sorted"
	B = 2
	C = 3
)

const (
	X = "x" // want "variable/constant declarations are not sorted"
	Y = "y"
	Z = "z"
)